src/main/java/uk/ac/ebi/spot/gwas/deposition/ingest/constants/IngestServiceConstants.java

```diff
@@ -24,6 +24,8 @@ public class IngestServiceConstants {
 
     public static final String PARAM_SUBMISSIONID = "submissionId";
 
+    public static final String PARAM_STUDY_TAG = "studyTag";
```

Comment:
Hi @sajo-ebi , Its better to keep it as kebab-case or snake_case, other endpoints in the ingest are kebab-case, I'll advice we use kebab-case for consistence i.e study-tag